### PR TITLE
Release Note: PCC Beta

### PR DESCRIPTION
**Pull request by tyherbert**
Release note for the content publisher beta announcement. 

Scheduled to be merged/deployed after 5 pm PT tonight (11/12/2024).

### CODE REVIEW

File: source/releasenotes/2024-11-12-content-publisher-beta.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Explore Pantheon Content Publisher—available now in Beta!
```
Comment (stevector) [inline source/releasenotes/2024-11-12-content-publisher-beta.md:2]:
```suggestion
title: Pantheon Content Publisher in private beta
```

Comment (scottbuscemi) [inline source/releasenotes/2024-11-12-content-publisher-beta.md:2]:
```suggestion
title: Explore Pantheon Content Publisher - available now in Beta!
```

IMO would be better to have spaces for easier legibility

File: source/releasenotes/2024-11-12-content-publisher-beta.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+published_date: "2024-11-12"
+categories: [New Feature, WordPress, Plugins, Drupal]
+---
+
+With Pantheon Content Publisher, your team can now publish content directly from Google Docs to Drupal, WordPress, or Next.js sites—quickly and seamlessly. Skip the extra steps and empower your team to keep content fresh and connected across platforms.
```
Comment (scottbuscemi) [inline source/releasenotes/2024-11-12-content-publisher-beta.md:7]: Is emdash usage normal on the docs / release notes?

File: source/releasenotes/2024-11-12-content-publisher-beta.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+* **Unified platform:** Draft, edit, and publish right from Google Docs.  
+* **Live previews**: See exactly how your content will look before it goes live.  
+* **Seamless collaboration:** Use comments and suggestions in Google Docs to streamline teamwork.  
+* **Publishing control**: Manage permissions and oversee all docs from a central dashboard.  
+* **Publish faster:** Faster publishing: Skip manual tasks with direct publishing from Google Docs.  
+* **Error reduction:** Avoid copy-paste mistakes and ensure content accuracy.  
+* **Efficient workflow:** Move smoothly from draft to published content without leaving Google Docs.
+
+Be among the first to experience Pantheon Content Publisher—request [Beta access](https://pantheon.io/platform/content-publisher) today\!
```
Comment (scottbuscemi) [inline source/releasenotes/2024-11-12-content-publisher-beta.md:21]:
```suggestion
Be among the first to experience Pantheon Content Publisher—request [Beta access](https://pantheon.io/platform/content-publisher) today!
```

File: source/releasenotes/2024-11-12-content-publisher-beta.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+With Pantheon Content Publisher, your team can now publish content directly from Google Docs to Drupal, WordPress, or Next.js sites—quickly and seamlessly. Skip the extra steps and empower your team to keep content fresh and connected across platforms.
+
+![ Alt Text](https://cdn.bfldr.com/MEM5087K/at/wcxzk54ps4ggw6rj4vqk7m89/platformoverview_featurestab_2400x1483-Web-ContentPublisher.gif?drupal-image-style=wide&width=1200&height=540&itok=b26FMzlt)
+
+**Here’s what you can expect:**
+
+* **Unified platform:** Draft, edit, and publish right from Google Docs.  
+* **Live previews**: See exactly how your content will look before it goes live.  
+* **Seamless collaboration:** Use comments and suggestions in Google Docs to streamline teamwork.  
+* **Publishing control**: Manage permissions and oversee all docs from a central dashboard.
```
Comment (scottbuscemi) [inline source/releasenotes/2024-11-12-content-publisher-beta.md:16]:
```suggestion
* **Publishing control:** Manage permissions and oversee all docs from a central dashboard.  
```

File: source/releasenotes/2024-11-12-content-publisher-beta.md (hunk 2 above)
Comment (IngridKwok) [reply source/releasenotes/2024-11-12-content-publisher-beta.md]: The emdashes are used correctly in this release notes - they're here to set off additional information. No spaces around emdashes. Do you find them odd?

**Review by stevector — APPROVED** [review]

**Pull request merged**